src/sail_lean_backend/pretty_print_lean.ml

```diff
@@ -266,7 +266,13 @@ and doc_typ ctx (Typ_aux (t, _) as typ) =
       parens (separate space [string "Result"; doc_typ ctx typ1; doc_typ ctx typ2])
   | Typ_var kid -> doc_kid ctx kid
   | Typ_app (id, args) -> parens (doc_id_ctor id ^^ space ^^ separate_map space (doc_typ_arg ctx `Only_relevant) args)
-  | Typ_exist (_, _, typ) -> doc_typ ctx typ
+  | Typ_exist (kids, _, typ) ->
+      let ctx =
+        List.fold_left
+          (fun ctx (KOpt_aux (KOpt_kind (_, kid), annot)) -> add_single_kid_id_rename ctx (Id_aux (Id "_", annot)) kid)
```

Comment:
Are you sure you want an `_` identifier to come out as a wildcard?  It feels rather fragile - an innocent change to provide a better encoding of identifiers could break this.

Reply:
I could put in a `_lean_wildcard` id and then change `fix_id` to print it as an underscore. Does that sound better?

Reply:
Sure.